### DISCUSSION TITLE
support for local tldr

### DISCUSSION THREAD

**Discussion by yanivpaz** [tldr-pages/tldr#8237] [Q&A] [answered]
Hi
1)Is there support for local TLDR which I dont want to share for external users?
2) where are the database files kept - is it all fetched from the git repo ?

**Comment by marchersimon** [comment]
Hi @yanivpaz 

With external users, do you mean users on the same machine? 
Which operating system do you use?

When you run `tldr --update` the whole tldr database is fetched from Github and then saved locally. Where it is saved depends on the client you use.

**Comment by navarroaxel** [comment, marked as answer]
The [tealdeer client](https://github.com/dbrgn/tealdeer) supports custom pages https://dbrgn.github.io/tealdeer/usage_custom_pages.html, the storage system is just Markdown files in a folder, but you can use Git and connect that configuration folder with a Git repo.
